internal/armada/repository/job.go

```diff
@@ -80,6 +83,9 @@ type JobRepository interface {
 	GetQueueActiveJobSets(queue string) ([]*api.JobSetInfo, error)
 	AddRetryAttempt(jobId string) error
 	GetNumberOfRetryAttempts(jobId string) (int, error)
+	StorePulsarSchedulerJobDetails(jobDetails []*schedulerobjects.PulsarSchedulerJobDetails) error
```

Comment:
These are the new functions for storing a minimal set of information in redis (just queue and jobset) for each job assigned to the pulsar scheduler